.node-version

```diff
@@ -0,0 +1 @@
+14.16.0
```

Comment:
These files should be removed. If they're created automatically by your environment or IDE you can add them to the `.gitignore` file.